project/Build.scala

```diff
@@ -1144,9 +1163,20 @@ object Build extends sbt.Build {
             val outFile = dir / "SourceMapTest.scala"
             IO.write(outFile, replaced.replace("0/*<testCount>*/", i.toString))
             Seq(outFile)
-          }
+          },
+
+        testFrameworks += new TestFramework("com.novocode.junit.JUnitFramework"),
```

Comment:
My point is, sbt itself adds that particular framework (`"com.novocode.junit.JUnitFramework"`) to *all* projects. So I think this is redundant, both here *and* in `ScalaJSJUnitPlugin`.